### PR TITLE
ci: separate the cephfs and rbd tests

### PR DESCRIPTION
**Pull request by Madhu-1**
As the maximum time allocated for the Travis CI is 50 minutes and the cephfs and rbd E2E takes around 45 Minutes to run completely. This is blocking us from adding more tests in E2E. splitting out the E2E will help us to run more tests for each driver as we will get 50 minutes for each Travis CI instance.

Added two new parameters for the e2e test to skip rbd and cephfs tests. This will help us to run more tests in Travis CI.
    

Signed-off-by: Madhu Rajanna <madhupr007@gmail.com>

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -95,25 +95,52 @@ jobs:
         - ./scripts/build-multi-arch-image.sh || travis_terminate 1;
 
     - stage: e2e testing
-      name: cephcsi with kube 1.16.9
+      name: cephcsi CephFS E2E with kubernetes v1.16.9
```
Comment (Yuggupta27) [inline .travis.yml:98]:
```suggestion
      name: cephcsi CephFS with kubernetes v1.16.9
```

Comment (Yuggupta27) [reply .travis.yml:98]: Since e2e testing is pre-mentioned, Will this be better?

File: .travis.yml [hunk 2]
```diff
@@ -95,25 +95,52 @@ jobs:
...
       script:
         - scripts/skip-doc-change.sh || travis_terminate 0;
         - make image-cephcsi || travis_terminate 1;
-        - scripts/travis-functest.sh v1.16.9 || travis_terminate 1;
+        - scripts/travis-functest.sh v1.16.9 --test-cephfs=true
+          --test-rbd=false || travis_terminate 1;
 
     - stage: e2e testing
-      name: cephcsi with kube 1.17.5
+      name: cephcsi RBD E2E with kubernetes v1.16.9
```
Comment (Yuggupta27) [inline .travis.yml:106]:
```suggestion
      name: cephcsi RBD with kubernetes v1.16.9
```

File: scripts/travis-helmtest.sh [hunk 3]
```diff
@@ -49,7 +49,7 @@ scripts/install-helm.sh up
 # install cephcsi helm charts
 scripts/install-helm.sh install-cephcsi ${NAMESPACE}
 # functional tests
-go test "${GO_TAGS}" github.com/ceph/ceph-csi/e2e -mod=vendor --deploy-timeout="${DEPLOY_TIMEOUT}" -timeout="${E2E_TIMEOUT}" --cephcsi-namespace=${NAMESPACE} --deploy-cephfs=false --deploy-rbd=false -v
+go test "${GO_TAGS}" github.com/ceph/ceph-csi/e2e -mod=vendor --deploy-timeout="${DEPLOY_TIMEOUT}" -timeout="${E2E_TIMEOUT}" --cephcsi-namespace=${NAMESPACE} --deploy-cephfs=false --deploy-rbd=false -v "${@:2}"
```
Comment (nixpanic) [inline scripts/travis-helmtest.sh:52]: Please use `shift` after line 6 where `${1}` is stored in a variable. After that, you can just pass `${@}` here to make clearer what is done.

Comment (Madhu-1) [reply scripts/travis-helmtest.sh]: done

**Comment by Madhu-1** [comment]
@humblec PTAL

**Pull request merged**